OP2MapImager.vcxproj

```diff
@@ -137,8 +139,12 @@ if $(ConfigurationName) == Release (
       <WarningLevel>Level3</WarningLevel>
       <Optimization>Disabled</Optimization>
       <PreprocessorDefinitions>_DEBUG;_CONSOLE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <AdditionalIncludeDirectories>OP2Utility\include;FreeImage\Dist\x64;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <RuntimeLibrary>MultiThreadedDebug</RuntimeLibrary>
```

Comment:
This property is also added in open branch #20. Maybe we should merge that branch. I was going to wait until we had finalized the change list, but maybe better to merge now.